.github/PULL_REQUEST_TEMPLATE/version_bump_template.md

```diff
@@ -0,0 +1,13 @@
+**Description**
+
+This PR is for the release version bump, the final step before tagging a beta release or creating a stable release branch.
+
+**Checklist**
+
+- [ ] `package.json` version string updated
+- [ ] `package-lock.json` version string updated (by running `npm install`)
+- [ ] changelog version string udpated
+
```

Comment:
Shall we also add a check for updating user manual URL?

Reply:
Good idea.